keyboards/scottokeebs/scotto34/keymaps/scotto/rules.mk

```diff
@@ -0,0 +1,3 @@
+TAP_DANCE_ENABLE = yes
```

Comment:
Would you be able to:
1. remove `RGBLIGHT_ENABLE` and `WS2812_DRIVER` definitions/lines
2. move `this file into `default` keymap directory